front/cypress/e2e/report_builder/widgets/posts_by_time_widget.cy.ts

```diff
@@ -120,21 +123,24 @@ describe('Report builder Posts By Time widget', () => {
     cy.wait(1000);
     cy.get('#e2e-content-builder-topbar-save').click();
     cy.wait('@saveReportLayout');
+    // Wait for reportLayout.attributes.craftjs_json update.
+    //
+    // The delete happens so quickly after save, that at the time
+    // `onNodesChange` is called, reportLayout.attributes.craftjs_json
+    // still has the initial value before save (empty).
+    // After the delete, the actual state is also empty.
+    // And so, the `saved` state is not properly updated.
+    // Also, see reactions_by_time_widget.cy.ts
+    cy.wait('@getReportLayout');
```

Comment:
Can we also do this for `comments_by_time_widget`? There we have the same issue.

Reply:
Done, thanks for pointing it out.

Maybe it's a good reason to try sharing more Cypress code.